R/get_cdc_data.R

```diff
@@ -0,0 +1,41 @@
+#' Retrieve CDC growth chart metadataset
+#'
+#' Retrieves CDC growth chart metadataset
+#'
+#' @param type Type of Chart
+#'
+#' A character is expected amongst: "height", "weight", "bmi"
+#'
+#' @return The appropriate growth chart from the CDC data
+#'
+#' @details
+#'
+#' Each data frame will be denoted as below:
+#' \describe{
+#'   \item{\code{SEX}}{double sex where 1 = male, 2 = female}
+#'   \item{\code{AGE}}{double age in months}
+#'   \item{\code{L}}{double Box-Cox transformation for normality}
+#'   \item{\code{M}}{double Median}
+#'   \item{\code{S}}{double Coefficient of variation}
+#' }
+#'
+#' from [cdc websites](https://www.cdc.gov/growthcharts/percentile_data_files.htm)
+#'
+#' @family metadata
```

Comment:
we need to update man/roxygen/meta.R too for this

Reply:
thanks @zdz2101 - we'll have to remember to come back to this later and add other categories once we add more functions

Reply:
i wonder here if we should be more generic such as `metadata = "Metadata"`?  like admiral has it, as what if we end up adding more than just CDC/WHO growth charts here over time.